website/content/docs/install/index.mdx

```diff
@@ -322,6 +334,35 @@ net.bridge.bridge-nf-call-iptables = 1
 
 </CodeBlockConfig>
 
+<h3>Install `consul-cni` for Consul service mesh</h3>
+
+If the `transparent_proxy` block is used for Consul Service Mesh, then the 
+`consul-cni` CNI Plugin must be installed on each client node to redirect inbound 
+and outbound traffic for services to the Envoy proxy. The following commands
+assume that the linux packages repositories have already been added using `apt` or 
+`yum` and that the reference CNI plugins have already been installed. 
+
+<Tabs>
+
+<Tab heading="CentOS/RHEL">
```

Comment:
Shouldn't these tabs match the tabs for the installation step above? (i.e. we should have Fedora for `dnf`, and Amazon Linux)